aten/src/ATen/core/type.cpp

```diff
@@ -41,10 +41,20 @@ std::ostream& operator<<(std::ostream & out, const Type & t) {
         } else {
           out << "*";
         }
-        if (has_valid_strides_info &&
-            type_verbosity() >= TypeVerbosity::TypeAndStride) {
-          out << ":" << *value->strides()[i];
+      }
+      if (has_valid_strides_info) {
```

Comment:
I might be missing something. Why is the condition here `has_valid_strides_info`? I thought the final `,` be printed as long as `*ndim > 0` regardless of `strides` condition.

Reply:
How about the case where there is complete strides info, but `type_verbosity` is lower than `TypeAndStride`? is `,` needed?

Reply:
Is this addressed?